tests/smoke/mon_test.go

```diff
@@ -29,21 +29,22 @@ import (
 func (suite *SmokeSuite) TestMonFailover() {
 	logger.Infof("Mon Failover Smoke Test")
 
+	namespace := "rook"
```

Comment:
let use defaultRookNamespace var instead of defining namespace, that way we can control defaultRookNamespace from common place.

Reply:
done